dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -335,7 +336,11 @@ export const Card = ({
 								/>
 							</AvatarContainer>
 						) : (
-							<img src={image.src} alt="" role="presentation" />
+							<FrontPicture
+								master={image.src}
```

Comment:
I realise that the `master` terminology wasn't introduced in this PR, and I'm also not quite sure whether this kind of usage shares the same etymology that's led to our moving away from `master` for git branches or if it has a different root.

Is this something that's been discussed before? If not, is there any reason why we couldn't switch to something like `primary` here, either now or in a future PR?

Reply:
I have heard it being discussed before and my understanding is that the name “master” when it relates to an original does not build on the terribly and inadequate analogy of slavery.

This being said, [Adobe have opted for the name “Source Graphic” instead in some of their tools](https://helpx.adobe.com/x-productkb/multi/terminology-changes-video-products.html). I think it might be worth brining up at the next discussion with Editorial Tools, as sharing terminology would be beneficial.